### ISSUE TITLE
<TresCanvas> results in continuous height growth of the <canvas>

### ISSUE THREAD

**Issue by AMP3RAGE** [Tresjs/nuxt#48]
### Describe the bug

I set up a brand new Nuxt3 project, added `@tresjs/nuxt` via npm and added it to modules section of **_nuxt.config.ts_** . I then added the `<TresCanvas>` to the **_app.vue_**. The resulting `<canvas ...>` in the DOM continuously grows in height... forever! What could be causing this, and how do I fix it? 

### Reproduction

https://stackblitz.com/edit/trescanvas-height-error?file=app.vue

### Steps to reproduce

Run `npm install` and then `npm run dev`. View the results using the url provided (typically [http://localhost:3000/](http://localhost:3000/)). When using Chrome you can see the height grow by opening the dev tools window and selecting the Elements and then hovering the mouse over the <body> element. If you drill down to the <canvas> element you'll see the height continuously increasing!!!

### System Info

```shell
No errors reported. But the size of the window is continuously increasing!!!
```

### Used Package Manager

npm

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/Tresjs/tres/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/Tresjs/tres/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://tresjs.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/tresjs/tres/issues) that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by alvarosabu** [comment]
Hi there, thanks for raising this, it might be solved by #44

**Comment by alvarosabu** [comment]
Hi @AMP3RAGE so the `TresCanvas` component will take the height of the parent component. So to make it work maker sure the parent has a height

https://stackblitz.com/edit/trescanvas-height-error-d9uacj?file=app.vue. 

Or you could set `window-size` on the canvas, adding style directly to the canvas would not work because the height is calculated based on the parent.
